feature/google/src/test/java/com/revenuecat/purchases/google/ParcelableTests.kt

```diff
@@ -9,9 +9,9 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import com.revenuecat.purchases.Package as RevenueCatPackage
 
-@RunWith(AndroidJUnit4::class)
-class ParcelableTests {
 //    TODOBC5: After using mocks for ProductDetails, these tests don't make much sense. Leaving for now.
```

Comment:
I don't think this comment makes sense because this has nothing to do with mocks

Reply:
Agreed, I don't know why I added this right now... But looking at it, I think we should absolutely have these tests.